sdk/openai/openai/sources/generated/src/api/operations.ts

```diff
@@ -30,90 +44,306 @@ import {
     .post({
       ...operationOptionsToRequestParameters(options),
-      body: { user: options?.user, model: options?.model, input: input },
-    });
+      body: {
+        file: uint8ArrayToString(body["file"], "base64"),
+        response_format: body["responseFormat"],
+        language: body["language"],
+        prompt: body["prompt"],
+        temperature: body["temperature"],
+        model: body["model"],
+      },
+    }) as StreamableMethod<
+    | GetAudioTranscriptionAsPlainText200Response
+    | GetAudioTranscriptionAsPlainTextDefaultResponse
+  >;
 }
 
-export async function _getEmbeddingsDeserialize(
-  result: GetEmbeddings200Response | GetEmbeddingsDefaultResponse
-): Promise<Embeddings> {
+export async function _getAudioTranscriptionAsPlainTextDeserialize(
+  result:
+    | GetAudioTranscriptionAsPlainText200Response
+    | GetAudioTranscriptionAsPlainTextDefaultResponse
+): Promise<string> {
+  if (isUnexpected(result)) {
+    throw result.body;
+  }
+
+  return result.body;
+}
+
+/**
+ * Gets transcribed text and associated metadata from provided spoken audio data. Audio will be transcribed in the
+ * written language corresponding to the language it was spoken in.
+ */
+export async function getAudioTranscriptionAsPlainText(
+  context: Client,
+  deploymentId: string,
+  body: AudioTranscriptionOptions,
+  options: GetAudioTranscriptionAsPlainTextOptions = { requestOptions: {} }
+): Promise<string> {
+  const result = await _getAudioTranscriptionAsPlainTextSend(
+    context,
+    deploymentId,
+    body,
+    options
+  );
+  return _getAudioTranscriptionAsPlainTextDeserialize(result);
+}
+
+export function _getAudioTranscriptionAsResponseObjectSend(
+  context: Client,
+  deploymentId: string,
+  body: AudioTranscriptionOptions,
+  options: GetAudioTranscriptionAsResponseObjectOptions = { requestOptions: {} }
+): StreamableMethod<
+  | GetAudioTranscriptionAsResponseObject200Response
+  | GetAudioTranscriptionAsResponseObjectDefaultResponse
+> {
+  return context
+    .path("/deployments/{deploymentId}/audio/transcriptions", deploymentId)
+    .post({
+      ...operationOptionsToRequestParameters(options),
+      contentType: (options.contentType as any) ?? "multipart/form-data",
+      body: {
+        file: uint8ArrayToString(body["file"], "base64"),
+        response_format: body["responseFormat"],
+        language: body["language"],
+        prompt: body["prompt"],
+        temperature: body["temperature"],
+        model: body["model"],
+      },
+    }) as StreamableMethod<
+    | GetAudioTranscriptionAsResponseObject200Response
+    | GetAudioTranscriptionAsResponseObjectDefaultResponse
+  >;
+}
+
+export async function _getAudioTranscriptionAsResponseObjectDeserialize(
+  result:
+    | GetAudioTranscriptionAsResponseObject200Response
+    | GetAudioTranscriptionAsResponseObjectDefaultResponse
+): Promise<AudioTranscription> {
   if (isUnexpected(result)) {
-    throw result.body.error;
```

Comment:
We'll need to make sure all throws statements are `throw result.body.error`.

Reply:
That issue currently is still open in the CodeGen [here](https://github.com/Azure/autorest.typescript/issues/1972), so we'll have to do mannual edits after we run the customization tool

Reply:
I am not opposed to doing it in the src folder but then we'll have to do the edits every time we run the customization tool as opposed to everytime we run code gen, pick your poison situation I guess 😅